src/hacbs/components/EnterpriseContractView/useEnterpriseContractPolicies.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { EnterpriseContractPolicies } from './types';
+
+export const useEnterpriseContractPolicies = (fileName: string) => {
+  const [contractPolicies, setContractPolicies] = React.useState<EnterpriseContractPolicies>();
+
+  React.useEffect(() => {
+    let canceled = false;
+
+    fetch(fileName).then(async (response) => {
+      const contractInfo: EnterpriseContractPolicies = await response.json();
+      if (!canceled) {
+        setContractPolicies(contractInfo);
+      }
+    });
+    return () => {
+      canceled = true;
+    };
+  }, [fileName]);
+
+  return contractPolicies;
```

Comment:
There should be a loading state.
Also useful if in the future we have a different mechanism for fetching this information.

Typical pattern is `[items, loading, error]`